gdrive

```diff
@@ -75,18 +77,18 @@ def upload(service, filepath):
 
     try:
         file_metadata = {'name': filename}
-        media = MediaFileUpload(filepath, mimetype=mime_type, resumable=True)
+        media = MediaFileUpload(filepath, mimetype=mime_type, resumable=True, chunksize=CHUNK_SIZE)
         created_file = service.files().create(body=file_metadata,
                                         media_body=media,
                                         fields='id')
         response = None
-        print("Uploading 0%", end='\r')
+        print("Uploading 0.0%", end='\r')
         while response is None:
             status, response = created_file.next_chunk()
             if status:
-                print("Uploading %d%%" % int(status.progress() * 100),end='\r')
+                print("Uploading %.1f%%" % float(status.progress() * 100), end='\r')
```

Comment:
I think I would prefer to have this as it was and having another information that would show the amount that has already being download. Something like `50% 50MB/100MB`. This second information could be updated more frequently. What do you think?